### PR TITLE
feat: add not-found page

### PR DESCRIPTION
**Pull request by floraSimpleDev**
adding not-found page in beequant-turbo.

Resolve CP-110

---

### CODE REVIEW

File: scripts/turbo.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ if [ -n "$CMD" ]; then
     if [ -n "$FILTER" ]; then
       case $FILTER in
       "client")
-        yarn turbo run dev --filter=$NAMESPACE/client --filter=$NAMESPACE/platform-api
+        yarn turbo run dev --filter=@beequant/client --filter=@beequant/platform-api
```
Comment (Chris-Wang) [inline scripts/turbo.sh:12]: why this introduced here?

**Review by Chris-Wang — CHANGES_REQUESTED** [review]

**Comment by floraSimpleDev** [comment]
Because I use the windows environment to run this project, I should modify
the turbo.sh file according to the tutorial of Norris.

**Comment by Chris-Wang** [comment]
for irrelevant changes, it shouldn't be included in the PR.

Also, does this change affect other non-windows users?

**Comment by floraSimpleDev** [comment]
Maybe not, cause non-windows users will run the project by the “yarn dev
client “ command, but I cannot use it successfully. If this changes
shouldn’t be here, I will fix it tomorrow.

**Pull request closed without merging**